site/docs/encoding/type.md

```diff
@@ -5,7 +5,12 @@ permalink: /docs/type.html
 ---
 
 If a field is specified, the channel definition **must** describe the encoded data's [type of measurement (level of measurement)](https://en.wikipedia.org/wiki/Level_of_measurement).
-The supported data types are: `"quantitative"`, `"temporal"`, `"ordinal"`, and `"nominal"`.
+The supported data types are: `"quantitative"`, `"temporal"`, `"ordinal"`, `"nominal"`, `"latitude"`, `"longitude"`, and `"geojson"`.
+
+{% include table.html props="type" source="FieldDef" %}
+
+**Note**:
```

Comment:
Thanks for adding this note.  This is definitely useful to add.  

A few thoughts:

- We should add it to `type.md` too

- For this page, I think maybe it's better to add this description to the JSDoc in the `FieldDef` interface so the explanation appears in the table.  Plus, Altair users can directly benefit from this explanation.

Reply:
Can you clarify what you mean? (also do you mean `types.md`?

Reply:
- There is type.md

- You can put the note in `FieldDef` interface in `fielddef.ts`  instead of here.